lib/db/index.js

```diff
@@ -108,6 +109,32 @@ function preClients() {
   }
 }
 
+function serviceClients() {
+  var clients = config.get('serviceClients');
+  if (clients && clients.length) {
+    logger.debug('serviceClients.loading', clients);
+
+    return P.all(clients.map(function(client) {
+      return exports.getClient(client.id).then(function(existing) {
+        if (existing) {
+          logger.verbose('seviceClients.existing', client);
+          return;
+        }
+
+        return exports.registerClient({
+          id: client.id,
+          name: client.name,
+          hashedSecret: encrypt.hash(unique.secret()),
```

Comment:
In other places we've used "0000000000000000000000000000000000000000000000000000000000000000" as the hashedSecret for clients that aren't supposed to use a secret.  Is that a pattern we should repeat for these service clients for consistency?